src/api/app/views/webui2/webui/project/_new_package_branch_modal.html.haml

```diff
@@ -0,0 +1,50 @@
+.modal.fade#new-package-branch-modal{ tabindex: '-1', role: 'dialog', aria: { labelledby: 'new-package-branch-modal-label', hidden: true } }
+  .modal-dialog.modal-lg{ role: 'document' }
+    .modal-content
+      .modal-header
+        %h5.modal-title#new-package-branch-modal-label
+          Add new package branch to #{project.name}
+        %button.close{ type: 'button', data: { dismiss: 'modal' }, aria: { label: 'close' } }
+          %span{ aria: { hidden: true } }
+            &times
+      .modal-body.ui-front
+        %p
```

Comment:
unfortunately this https://github.com/openSUSE/open-build-service/blob/a2b57207919d2dc70aa765d491c6bf8449407bab/src/api/app/assets/stylesheets/webui2/modals.scss#L8 breaks the layout a little bit, because it wraps the lines at the places where it got wraped in the source code. @Ana06 do you think we can use ```white-space: normal``` instead or should i just overwrite it for my modal? :)

Reply:
okay its resolved through https://github.com/openSUSE/open-build-service/commit/f8981835987ee66beb7788897d902007cd1f234f